src/screens/Introduction/Slide/HistorySlide.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { translate } from "../../../i18n";
+import { I18_NAMESPACE } from "../constants";
+import SlideView from "./SlideView";
+
+const image = require("../../../assets/introduction/history_screen.png");
+
+const HistorySlide = () => (
+  <SlideView
+    image={image}
+    title={translate(`${I18_NAMESPACE}historyScreenTitle`)}
+    description={translate(`${I18_NAMESPACE}historyScreenText`)}
+  />
+);
```

Comment:
how about smth like:

```
const slidesInfo = [
{ image: require("../../../assets/introduction/history_screen.png"), title: translate(`${I18_NAMESPACE}historyScreenTitle`), description: translate(`${I18_NAMESPACE}historyScreenText`) },
....
];

slidesInfo.map(slideInfo => <SlideView {...slideInfo} />);
```

instead of 4 different components
?

Reply:
Agree))) Its look very strange))